devices/src/gic.rs

```diff
@@ -23,6 +26,22 @@ type Result<T> = result::Result<T, Error>;
 pub const IRQ_LEGACY_BASE: usize = layout::IRQ_BASE as usize;
 pub const IRQ_LEGACY_COUNT: usize = 32;
 
+/// Default config implied by layout::*
+pub fn create_vgic_config(vcpu_count: u64) -> VgicConfig {
```

Comment:
Okay. Move this to `Gic::create_default_config()`

Reply:
ok!